scripts/jenkins

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -e
+set -o
+
+MASTER_BRANCH="${MASTER_BRANCH:-origin/master}"
+
+HS_TOP="./src/main/haskell/kore"
+HS_SOURCE_DIRS=("$HS_TOP/src" "$HS_TOP/app" "$HS_TOP/test" "$HS_TOP/bench")
+
+STACK_OPTS="${STACK_OPTS:---pedantic}"
+STACK_BUILD_OPTS="$STACK_OPTS --coverage --no-run-tests --no-run-benchmarks"
+STACK_HADDOCK_OPTS="$STACK_OPTS --no-run-tests --no-run-benchmarks"
+STACK_TEST_OPTS="$STACK_OPTS --no-run-benchmarks"
+
+COVERAGE_REPORT=".build/coverage_report"
+HASKELL_DOCUMENTATION=".build/haskell_documentation"
+
+notif() { echo "== JENKINS: $@" >&2 ; }
+fatal() { notif "FATAL: $@" ; exit 1 ; }
+
+notif "Cleaning ..."
+# ------------------
+stack clean
+find . -name '*.tix' -delete
+make --directory src/main/k/working clean -j8
+
+notif "Check ..."
+# ---------------
+./scripts/git-assert-clean.sh \
+    || fatal "Please commit your changes!"
+
+./scripts/git-rebased-on.sh "$(git rev-parse $MASTER_BRANCH)" --linear \
+    || fatal "Please rebase your branch on 'master'!"
+
+notif "Stylish ..."
+# -----------------
+for hs_file in $(find ${HS_SOURCE_DIRS[@]} -name '*.hs' -o -name '*.hs-boot'); do
```

Comment:
I think `find -exec` returns a non-zero exit code if `stylish-haskell` fails on any file; this should do the same.

Reply:
I have `set -e` at the top, which means that if at any point in the process a process returns a non-zero exit code the whole script will exit with that code.